packages/network-of-terms-catalog/catalog/datasets/picot.jsonld

```diff
@@ -0,0 +1,69 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "https://terms.personsincontext.org/ThesaurusHistorischePersoonsgegevens",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Thesaurus Historische Persoonsgegevens"
+    }
+  ],
+  "alternateName": [
+    {
+      "@language": "en",
+      "@value": "PiCoT"
+    }
+  ],
+  "genre": [
+    {
+      "@id": "https://data.cultureelerfgoed.nl/termennetwerk/onderwerpen/Actoren-en-organismen"
+    }
+  ],
+  "creator": [
+    {
+      "@id": "https://www.cbg.nl"
+    }
+  ],
+  "url": [
+    "https://terms.personsincontext.org/ThesaurusHistorischePersoonsgegevens/"
+  ],
+  "mainEntityOfPage": [
+    "https://terms.personsincontext.org/ThesaurusHistorischePersoonsgegevens"
+  ],
+  "description": [
+    {
+      "@language": "nl",
+      "@value": "Rollen voor het beschrijven van personen in historische documenten"
```

Comment:
@rschalkrce @pwoltjer Not just roles but also source types, though. Can you provide a better description?

Reply:
I will pass this on to Pieter